README.md

```diff
@@ -36,8 +67,6 @@ Conditions:
 1. You must do this in a language that you have not programmed in before.
 2. You must not use a library that provides a `reverse` method or function.
 
-That's it!
-
 Here are some of the languages (in no particular order) that we use at Trike on the side to help us
 solve problems.
 [Rust](https://www.ruby-lang.org/en/), [Swift](https://swift.org), [Haskell](haskell.org),
```

Comment:
Underhanded rubyist propaganda, I tells ya!

Also "haskell.org" should be a proper URL.